src/sentry/snuba/sessions_v2.py

```diff
@@ -217,6 +217,7 @@ def get_keys_for_row(self, row):
 }
 
 CONDITION_COLUMNS = ["project", "environment", "release"]
+FILTER_KEY_COLUMNS = ["project_id"]
```

Comment:
"project_id" is the only field that is _supposed_ to end up in `filter_keys`, as far as I can tell.